src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD002UseJtfRunAnalyzerTests.cs

```diff
@@ -42,6 +42,68 @@ class Test {
             await Verify.VerifyCodeFixAsync(test, expected, withFix);
         }
 
+        [Fact]
+        public async Task TaskWaitAnyShouldReportWarning()
+        {
+            var test = @"
+using System;
+using System.Threading.Tasks;
+
+class Test {
+    void F() {
+        var task1 = Task.Run(() => {});
+        var task2 = Task.Run(() => {});
+        Task.WaitAny(task1, task2);
```

Comment:
This method returns a value. `Task.WaitAny(Task[])` returns an `int`, but `Task.WhenAny(Task[])` returns a `Task`. So in at least some cases a simple code fix implementation would introduce a compile error. What is the plan for handling that?
Should we avoid offering a code fix unless the InvocationExpression is directly within an ExpressionStatement?

Reply:
It seems like we can leave this like it is for now, and perhaps tweak it in the future.